README.md

```diff
@@ -1,216 +1,265 @@
+   
+   ```text
+   Environment diagnose (to see all details, use -v option):
+   [✓] Operation System
+   [✓] Java
+   [✓] Android Studio
+   [✓] Xcode
+   [✓] Cocoapods
+   
+   Conclusion:
+     ✓ Your system is ready for Kotlin Multiplatform Mobile Development!
+   ```
+   
+   Otherwise, KDoctor will highlight which parts of your setup still need configuration and suggest how to fix them.
 
-Switch to the Project View to see all files and targets belonging to the project.
+## Examine the project structure
+
+Open the project in Android Studio and switch the view from **Android** to **Project** to see all the files and targets belonging to the project:
 
 <img src="readme_images/open_project_view.png" height="300px">
 
 Your Compose Multiplatform project includes three modules:
 
-### `shared`
-This Kotlin module that contains the logic common for both Android and iOS applications – the code you share between platforms.
+### shared
+
+This is a Kotlin module that contains the logic common for both Android and iOS applications, the code you share between platforms.
+
+This `shared` module is also where you write your Compose Multiplatform code.
+In `shared/src/commonMain/kotlin/App.kt`, you can find the shared root `@Composable` function for your app.
 
-**This is also where you will write your Compose Multiplatform code**.
+It uses Gradle as the build system. You can add dependencies and change settings in `shared/build.gradle.kts`.
+The shared module builds into an Android library and an iOS framework.
 
-The shared root `@Composable` function for your app lives in `shared/src/commonMain/kotlin/App.kt`.
+### androidApp
 
-`shared` uses Gradle as the build system. You can add dependencies and change settings in `shared/build.gradle.kts`. The shared module builds into an Android library and an iOS framework.
+This is a Kotlin module that builds into an Android application. It uses Gradle as the build system.
+The `androidApp` module depends on and uses the shared module as a regular Android library.
 
-### `androidApp`
+### iosApp
 
-This Kotlin module that builds into an Android application. It uses Gradle as the build system. The androidApp module depends on and uses the shared module as a regular Android library.
+This is an Xcode project that builds into an iOS application.
+It depends on and uses the shared module as a CocoaPods dependency.
 
-### `iosApp`
-This is the Xcode project that builds into an iOS application. It depends on and uses the shared module as a CocoaPods dependency.
+## Run your application
 
-## Running your application
+### On Android
 
-## Android
 To run your application on an Android emulator:
 
-- Create an [Android virtual device](https://developer.android.com/studio/run/managing-avds#createavd).
-- Select the `androidApp` run configuration.
-- Select your target device and press **Run**.
+1. Create an [Android virtual device](https://developer.android.com/studio/run/managing-avds#createavd).
```

Comment:
Is this step always necessary? When I tried, I didn't create any. Maybe it was created beforehand when I created a KMM project.

Reply:
If it is, can we open the link in another tab?

Reply:
It is necessary. Otherwise, the run configuration is unavailable
People who already have a virtual device in the list will have no problem of running the app

Reply:
Seems we're out of luck. "Does not include the target attribute"

Reply:
Technically, it's "ensure you have an Android virtual device – create one if you don't have one". Not sure if it needs to be pointed out explicitly though – those who have done Android Dev before will understand that they can just use their AVD

Reply:
Rephrased the first step to make this idea clearer

Reply:
Sounds good!